models/affiliation_model.js

```diff
@@ -0,0 +1,36 @@
+const mongoose = require("mongoose");
+const diffHistory = require("mongoose-diff-history/diffHistory");
+
+const { Schema } = mongoose;
+
+const affiliationSchema = new Schema(
+  {
+    // ambassadorName: {
+    //   type: String,
+    //   required: true,
+    // },
```

Comment:
@dev-scott remove this it won't be useful as the ambassadorId will already serve as connection in case we want to retrieve the ambassador's details.

Reply:
Ok